File: Data Structures & Algorithms/duplicate-integer/submission-3.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        const uniqueSet = new Set();
        nums.forEach((num) => uniqueSet.add(num))

        if (nums.length === uniqueSet.size) {
            return false;
        }
        return true;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-3.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        d = {}
        for num in nums:
            d[num] = d.get(num, 0) + 1
        d = dict(sorted(d.keys(), key=lambda x: x[1], reverse=True))
        print(d)

        return list(d.keys())[:k]
        